.github/workflows/social-connect.yml

```diff
@@ -41,7 +41,8 @@ jobs:
     # runs-on: ubuntu-latest
     runs-on: ['self-hosted', 'org', '8-cpu']
     container:
-      image: us-west1-docker.pkg.dev/devopsre/actions-runner-controller/celo-monorepo:node18
+      image: us-west1-docker.pkg.dev/devopsre/actions-runner-controller/celo-monorepo:node22
+      options: --user root
```

Comment:
@alvarof2 Have you ever experienced this? Is this a problem with the monorepo docker image?

Reply:
Not that I'm aware of. 
I know that @jcortejoso and @pputman-clabs enabled the use of the StepSecurity Hardened Runner Arc for self-hosted runners. Could this be related to that?